qa/evil-tests/src/test/java/org/elasticsearch/common/logging/EvilLoggerTests.java

```diff
@@ -191,49 +192,6 @@ public void testConcurrentDeprecationLogger() throws IOException, UserException,
 
     }
 
-    public void testDeprecationLoggerMaybeLog() throws IOException, UserException {
```

Comment:
I removed this test because the rate-limiting functionality now lives in a dedicated Filter.